hazelcast-spring/src/main/java/com/hazelcast/spring/AbstractHazelcastBeanDefinitionParser.java

```diff
@@ -398,7 +398,7 @@ protected BeanDefinition getEvictionConfig(final Node node) {
             }
 
             try {
-                EvictionConfigHelper.checkEvictionConfig(evictionPolicyValue, comparatorClassNameValue, comparatorBean);
+                checkEvictionConfig(evictionPolicyValue, comparatorClassNameValue, comparatorBean, false);
```

Comment:
When I read the method call.. I have difficulties with true and false in lot of cases.